myhpi/templates/login.html

```diff
@@ -0,0 +1,9 @@
+{% if user.is_authenticated %}
```

Comment:
what does `is_authenticated` mean? for me it reads like: "I can login if I am already authenticated" ?

Reply:
Exactly, but this is a really good catch because the template is not used yet 😅

Reply:
what is it for then?

Reply:
It should be used, we just forgot to finish this